sdk/python/src/dagger/api/gen_sync.py

```diff
@@ -569,11 +569,11 @@ def with_new_directory(self, path: str) -> "Directory":
         _ctx = self._select("withNewDirectory", _args)
         return Directory(_ctx)
 
-    def with_new_file(self, path: str, contents: str | None = None) -> "Directory":
```

Comment:
/cc @helderco I think `gen_sync.py` is missing from `.gitattributes` :)

Reply:
Indeed 😊

Reply:
@aluzzardi Quick update: https://github.com/dagger/dagger/pull/3926